types/rc-select/index.d.ts

```diff
@@ -20,41 +20,55 @@ export {
 
 declare namespace RcSelect {
     interface SelectProps {
-        className?: string;
-        prefixCls?: string;
+        allowClear?: boolean;
         animation?: string;
-        transitionName?: string;
         choiceTransitionName?: string;
-        dropdownMatchSelectWidth?: boolean;
+        className?: string;
+        combobox?: boolean;
+        defaultActiveFirstOption?: boolean;
+        defaultLabel?: string | Array<string>;
+        defaultValue?: string | Array<string>;
+        disabled?: boolean;
         dropdownClassName?: string;
-        dropdownStyle?: { [key: string]: string };
+        dropdownMatchSelectWidth?: boolean;
         dropdownMenuStyle?: { [key: string]: string };
-        notFoundContent?: string;
-        showSearch?: boolean;
-        allowClear?: boolean;
-        tags?: boolean;
+        dropdownStyle?: { [key: string]: string };
+        filterOption?: boolean;
+        getPopupContainer?: (trigger: Node) => Node;
+        getInputElement?: () => Node;
+        id?: string;
+        labelInValue?: boolean;
+        maxTagCount?: number;
+        maxTagPlaceholder?: React.ReactNode | Function;
         maxTagTextLength?: number;
-        combobox?: boolean;
         multiple?: boolean;
-        disabled?: boolean;
-        filterOption?: boolean;
-        optionFilterProp?: string;
-        optionLabelProp?: string;
-        defaultValue?: string | Array<string>;
-        value?: string | Array<string>;
+        notFoundContent?: string;
+        onBlur?: Function;
```

Comment:
It's not good practice to use the `Function` type as discussed [here](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes). Can you be more specific?